src/pybind/mgr/dashboard/model/nvmeof.py

```diff
@@ -1,8 +1,8 @@
-from typing import List, NamedTuple, Optional
+from typing import List, NamedTuple, Optional, Annotated
 
 
 class GatewayInfo(NamedTuple):
-    cli_version: str
+    cli_version: Annotated[str, 'drop']
```

Comment:
Instead of adding a `str`, you can use [Enum flags](https://docs.python.org/3/library/enum.html#enum.Flag):
```py
from enum import Flag, auto

class CLI_FLAGS(Flag):
    DROP = auto()
    EXCLUSIVE_LIST = auto()
    SIZE = auto()
```
Ok, now I see that some of the "flags" have params, so not really flags... You could define several taps of "modifiers": flags, objects, like:
```py
    rw_ios_per_second: Annotated[int, CLI_FLAGS.WHATEVER, CLI_Header('R/W IOs / second'), ....]
```

Reply:
It sounds 100 times better than what I did haha! will do it!